plugins/CorePluginsAdmin/lang/en.json

```diff
@@ -52,6 +52,7 @@
         "PluginNotDownloadablePaidReason": "Possible reasons are an expired or exceeded license.",
         "PluginRequiresInternet": "This plugin requires a working internet connection. Please check your config value for `enable_internet_features` if you want to use this feature.",
         "PluginActivated": "Plugin activated",
+        "PluginFreeTrialStarted": "<strong>Your 30-day free trial has started<\/strong> for %s.",
```

Comment:
We should avoid using HTML in translations. They often break when they are translated, as weblate automatically converts html special chars to entities unless the translation is marked correctly on weblate.